File: app/src/main/java/com/young/simpledict/filecache/request/GetFileResponse.java
package com.young.simpledict.filecache.request;

import com.young.simpledict.service.event.BaseEvent;

import java.io.File;

/**
 * Author: landerlyoung
 * Date:   2014-10-29
 * Time:   12:02
 * Life with passion. Code with creativity!
 */
public class GetFileResponse extends BaseEvent {
    public File resultFile;
}

File: app/src/main/java/com/young/simpledict/filecache/request/GetFileResponse.kt
package com.young.simpledict.filecache.request

import com.young.simpledict.service.event.BaseEvent

import java.io.File

/**
 * Author: landerlyang
 * Date:   2014-10-29
 * Time:   12:02
 * Life with passion. Code with creativity!
 */
class GetFileResponse(eventCode: Any, val resultFile: File?) : BaseEvent(eventCode)
